prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -709,29 +719,17 @@ def construct_and_run_prompt(
         context=context,
     )
     return run_completion(
-        llm_helper=llm_helper,
-        llm_li=llm_li,
-        prompt=prompt,
-        adapter_instance_id=output[PSKeys.LLM],
+        llm,
+        prompt,
```

Comment:
Can we pass this as named args instead of positional args for better readability?

Reply:
DOne